### ISSUE TITLE
Cannot import files when using a resource query

### ISSUE THREAD

**Issue by dimitarnestorov** [peerigon/extract-loader#53]
![image](https://user-images.githubusercontent.com/8790386/45590214-4a44cf80-b93c-11e8-9d7a-20c227586155.png)

If I omit `?external` it imports the image but inline, forcing me to use data URIs.

I created a repository showcasing the issue: https://github.com/dimitarnestorov/sass-loader-resource-query-test

At first I thought the problem was in sass-loader so I opened an [issue](https://github.com/webpack-contrib/sass-loader/issues/615) there but it was later discovered that extract-loader was the culprit: https://github.com/webpack-contrib/sass-loader/issues/615#issuecomment-442823763

**Comment by ckrjq** [comment]
I encountered the same problem when trying to include `@mdi/font`, which also use a query-param:

sass file:
```
$mdi-font-path: '~@mdi/font/fonts';
@import "~@mdi/font/scss/materialdesignicons";
```

webpack config (relevant part):
```
            {
                test: /\.s[ac]ss$/,
                exclude: /node_modules/,
                use: [
                    {
                        loader: 'file-loader',
                        options: {
                            name: '[name].[ext]',
                        }
                    },
                    {
                        loader: 'extract-loader',
                    },
                    'css-loader',
                    'sass-loader'
                ],
            },
```

Error message:

`Error: Cannot find module '@mdi/font/fonts/materialdesignicons-webfont.eot?v=3.2.89' from '/srv/s/website-poc/src/styles'`

**Comment by dimitarnestorov** [comment]
@ckrjq can you test my pull request?

In your `package.json` change `extract-loader` to be `dimitarnestorov/extract-loader#be04fb0496be7ccd946618c423366e9611ff45af` like so:
![image](https://user-images.githubusercontent.com/8790386/49814774-044cab80-fd73-11e8-9b32-b883b9e33f6a.png)

Then just run `yarn`/`npm install` to install that version and run webpack.

**Comment by ckrjq** [comment]
Tested. This fixes my issue.

**Comment by arturaum** [comment]
@dimitarnestorov thats works! Useful!

**Comment by jannikkeye** [comment]
Fixed in #66
